kratos/sources/kratos_parameters.cpp

```diff
@@ -228,6 +228,14 @@ Parameters::Parameters(Parameters const& rOther)
 /***********************************************************************************/
 /***********************************************************************************/
 
+Parameters::Parameters(Parameters&& rOther) : Parameters()
+{
+    swap(rOther);
```

Comment:
if this is custom then maybe a different name would be better to not confuse it with the one in the standard ...?